services/fdb/server/fdbcli.go

```diff
@@ -865,13 +1064,44 @@ func parseFDBCLISleep(req *pb.FDBCLISleep) ([]string, []captureLogs, error) {
 	return args, nil, nil
 }
 
+func parseFDBCLISnapshot(req *pb.FDBCLISnapshot) ([]string, []captureLogs, error) {
+	args := []string{"snapshot"}
+
+	if req.Command == "" {
+		return nil, nil, status.Error(codes.InvalidArgument, "snapshot requires command to be filled in")
+	}
+	args = append(args, req.Command)
+	args = append(args, req.Options...)
+	return args, nil, nil
+}
+
 func parseFDBCLIStatus(req *pb.FDBCLIStatus) ([]string, []captureLogs, error) {
 	args := []string{"status"}
 
 	args = stringFlag(args, req.Style, "")
 	return args, nil, nil
 }
 
+func parseFDBCLISuspend(req *pb.FDBCLISuspend) ([]string, []captureLogs, error) {
+	args := []string{"suspend"}
+
+	if req.Request == nil {
+		return nil, nil, status.Error(codes.InvalidArgument, "suspend requires request to be filled in")
+	}
+	switch req.Request.(type) {
```

Comment:
do you want to return an error in the 'default' case here? might make it more obvious to debug if we added a new unhandled case